### DISCUSSION TITLE
Quasar CLI Installer (Yarn or NPM)

### DISCUSSION THREAD

**Discussion by susansiow** [quasarframework/quasar#13263] [CLI - BEX mode] [answered]
I am preparing the upgrading work from Quasar v1 to v2 and I was not able to retrieve the global installer (yarn or npm) of @quasar/cli with the following commands:

`yarn global list`
`npm list -g`

The following is my local version list:

- NodeJs - 16.14.2
- NPM - 8.5.0
- yarn - 1.22.17
- @quasar/cli - 1.1.3
- quasar - 1.14.5

Besides the search of the Quasar CLI installer info (for upgrading purpose), I would also like to know the minimal version of it for Quasar v2, thank you!

**Comment by MilosPaunovic** [comment, marked as answer]
You could run `npm i -g @quasar/cli` to get the latest version of it; and it's always a good idea to keep your dependencies up-to-date, so, latest available is the one you want.

**Reply by susansiow** [reply]
Thanks for your kind response, is there a difference between the application of `npm i -g @quasar/cli` and `npm upgrade -g @quasar/cli`? Do these two commands provide the same upgrading purpose?

**Reply by MilosPaunovic** [reply]
They have the same end result.

**Reply by susansiow** [reply]
Thank you!
